js/src/sdk/client/schemas.gen.ts

```diff
@@ -2672,6 +2689,21 @@ export const $GetTriggerResponseDTO = {
     required: ['status']
 } as const;
 
+export const $WehbookNewFormatDTO = {
+    properties: {
+        id: {
+            type: 'string',
+            description: 'The ID of the project'
+        },
+        isNewWebhook: {
+            type: 'boolean',
+            description: 'The callback URL if found'
```

Comment:
The description for `isNewWebhook` in `WehbookNewFormatDTO` is incorrect. It says "The callback URL if found" but the field is actually a boolean indicating whether to use the new webhook format.